front/app/modules/commercial/verification/citizen/components/VerificationMethods.tsx

```diff
@@ -23,7 +23,7 @@ import styled from 'styled-components';
 import { colors, fontSizes, media } from 'utils/styleUtils';
 
 // typings
-import { IVerificationMethod } from 'services/verificationMethods';
+import { TVerificationMethodName } from 'services/verificationMethods';
```

Comment:
Some renaming (T stands for type, whereas I stands for interface, which was wrong).